### PR TITLE
Add test cases ids

### PR DESCRIPTION
**Pull request by ILpinto**
Signed-off-by: ipinto <ipinto@redhat.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/migration_test.go [hunk 1]
```diff
@@ -1666,8 +1666,8 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 				}
 
 				By("waiting until the VMIs are ready")
-				for _, vmi := range vmis {
-					tests.WaitForSuccessfulVMIStartWithTimeout(vmi, 180)
+				for _, vmi := range vmis {It(It(It(It(
```
Comment (davidvossel) [inline tests/migration_test.go:1669]: whoa, something weird is going on here.

File: tests/vmi_lifecycle_test.go [hunk 2]
```diff
@@ -1556,7 +1556,7 @@ var _ = Describe("[rfe_id:273][crit:high][vendor:cnv-qe@redhat.com][level:compon
 				pod := tests.GetRunningPodByVirtualMachineInstance(vmi, vmi.Namespace)
 				Expect(pod).NotTo(BeNil())
 				output, err := tests.ExecuteCommandOnPod(
-					virtClient,
+					virtClient,qq
```
Comment (davidvossel) [inline tests/vmi_lifecycle_test.go:1559]: what's this change for?

File: go.mod [hunk 3]
```diff
@@ -111,3 +111,5 @@ replace (
 
 	kubevirt.io/client-go => ./staging/src/kubevirt.io/client-go
 )
+
+go 1.13
```
Comment (davidvossel) [inline go.mod:115]: can we remove this from the commit

File: tests/console_test.go [hunk 4]
```diff
@@ -255,7 +255,7 @@ var _ = Describe("[rfe_id:127][posneg:negative][crit:medium][vendor:cnv-qe@redha
 				Expect(len(runningVMISpec.Devices.Consoles)).To(Equal(0), "should not have any virtio console for serial consoles")
 			})
 
-			It("should not connect to the serial console", func() {
+			It("[test_id:4119]should not connect to the serial console", func() {
```
Comment (kbidarkar) [inline tests/console_test.go:258]: this one needs to be 4118

File: tests/vm_test.go [hunk 5]
```diff
@@ -166,7 +166,7 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 			Expect(createdVM.Spec.Template.Spec.Domain.Machine.Type).To(Equal(testingMachineType))
 		})
 
-		It("should keep the supplied MachineType when created", func() {
+		It("[test_id:3312]should keep the supplied MachineType when created", func() {
```
Comment (kbidarkar) [inline tests/vm_test.go:169]: The test_id here and at line 157 are the same. Tried looking at polarion, could not find at a quick glance. May need to revisit this.

File: tests/vm_test.go [hunk 6]
```diff
@@ -154,7 +154,7 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 			return newVM
 		}
 
-		It("should set the default MachineType when created without explicit value", func() {
+		It("[test_id:3312]should set the default MachineType when created without explicit value", func() {
```
Comment (kbidarkar) [inline tests/vm_test.go:157]: The test_id here and at line 169 are the same.

File: tests/vmi_configuration_test.go [hunk 7]
```diff
@@ -1322,7 +1322,7 @@ var _ = Describe("Configurations", func() {
 				Expect(domXML).To(ContainSubstring("<entry name='serial'>4b2f5496-f3a3-460b-a375-168223f68845</entry>"), "Should have serial-number present")
 			})
 
-			It("should not have serial-number set when not present", func() {
+			It("[test_id:3905]should not have serial-number set when not present", func() {
```
Comment (kbidarkar) [inline tests/vmi_configuration_test.go:1325]: This one appears to be test_id:3122

From polarion this test_id appears to be with the name, `should have serial-number set when not present`

File: tests/console_test.go (hunk 4 above)
Comment (ILpinto) [reply tests/console_test.go:258]: Fixed

File: tests/vm_test.go (hunk 6 above)
Comment (ILpinto) [reply tests/vm_test.go:157]: Fixed

File: tests/vm_test.go (hunk 5 above)
Comment (ILpinto) [reply tests/vm_test.go:169]: Fixed

File: tests/vmi_configuration_test.go (hunk 7 above)
Comment (ILpinto) [reply tests/vmi_configuration_test.go:1325]: Fixed

File: go.mod (hunk 3 above)
Comment (kbidarkar) [reply go.mod]: I guess this is automatically added by vscode.

**Comment by kbidarkar** [comment]
/lgtm 

Apart from removing the auto addition in go.mod by maybe vscode

**Comment by ILpinto** [comment]
@kbidarkar 
go.mod doesn't no exists in the last commit

**Comment by davidvossel** [comment]
/approve

needs rebase though

**Comment by ILpinto** [comment]
Close this PR new PR created with test ids
https://github.com/kubevirt/kubevirt/pull/3380

**Pull request closed without merging**